docs/getting_started/installation.md

```diff
@@ -75,6 +75,12 @@ N rows in set (0.00 sec)
 
 ~~~
 
+!!! Note
+    MySQL v5.7 uses a default transaction isolation level of REPEATABLE-READ. You'll need to change this to READ-COMMITTED so that iRODS operates properly. This setting needs to be set globally. One way to achieve this is by starting MySQL with --transaction-isolation=READ-COMMITTED. For more information, see https://dev.mysql.com/doc/refman/5.7/en/innodb-transaction-isolation-levels.html.
+    
+!!! Note
+    MySQL v5.7 queries operate in a case-insensitive manner by default. iRODS requires that data be handled in a case-sensitive way. One way to achieve this is by starting MySQL with --character-set-server=utf8 and --collation-server=utf8_bin. However, due to the design of the iRODS database tables, the database may not start. As an alternative, you can set --collation-server=latin_general_cs (MySQL defaults to a latin1 character-set). For more information, see https://dev.mysql.com/doc/refman/5.7/en/charset-applications.html.
```

Comment:
use markdown links...  with the `[linktext](url)` notation... just keep the url as the linktext, that's fine.

and put backticks around the options in these two notes (like `--character-set-server=utf8`)